### DISCUSSION TITLE
Struggling to change layout of the posts on home and in my reviews.

### DISCUSSION THREAD

**Discussion by add1ct3dd** [gethinode/hinode#1117] [General]
Hi,

I'm brand new to hugo and hinode so I'm probably going about this the wrong way but I've tried everything I could think of and I'm coming up blank and I'd love someone to point me in the right direction.

I've successfully limited the scope of my homepage to show only my 'reviews' and 'posts' via this;

`[home]
    sections = ["reviews","posts"]`

However I'd love to display these posts in a list, full width one below another but I'm struggling to work out where and what I need to change the layout to "list" and set the respective "cols" to do this. I'm also seeing that on my 'reviews' and 'posts' pages, they are also formatted as cards which I'd like to change, but also unsure how I go about changing this, can anyone point me in the right direction as to what I actually need to change to do this?

I've ran through the docs a bunch of times but I think I'm getting confused over how the sections, posts, layout and content all fit into these properties. I've never used Hugo or indeed Hinode prior to this so it's a relatively big learning curve.

Appreciate any help, thanks!

**Comment by markdumay** [comment]
hi @add1ct3dd, I've added a new guide on how to customize the appearance of a Hinode site: https://gethinode.com/guides/custom-appearance/ Hope this helps! Feel free to share your thoughts / feedback.

**Reply by add1ct3dd** [reply]
That's awesome thanks @markdumay I'll give it a run through in the next few days and let you know, but from a quick glance it looks fantastic thank you so much!
